### ISSUE TITLE
Search results different on local with SQLite than cloud environments

### ISSUE THREAD

**Issue by amirkhan81** [skttl/umbraco-fulltextsearch8#132]
Umbraco version: 14.3.1
Full Text Search version: 5

I've implemented full text search on a cloud project, the search is working on local, dev and live but on local its returning less results than the backoffice external examine search results.

For example if I search "learning" on local, it only returns a page titled "learning", if i do the same on either of the cloud environments it returns all of the pages with the word "learning" in the content as expected. Searching for "learn" returns nothing at all on local.

Is there some setting related to the local SQLite database I'm using? I'm running this on a mac. The search works, but the results are way less than the other environments.

**Comment by skttl** [comment]
Hi @amirkhan81 

FullTextSearch should add some content in your External examine index, eg. in FullTextContent_[culture] fields (depending on your appsettings). Can you verify with the examine dashboard, that the content is there?

If it's not, then try reindexing, by rightclicking a node in the content tree, and select the reindex option.

**Comment by amirkhan81** [comment]
@skttl I see the FullTextContent field in the backoffice search, but its empty on local and populated on dev / live. Note that I didn't update any of the settings for full text search, just did the install. I reindexed both from the content side and the examine dashboard as well.

**Comment by MikeB1337** [comment]
Hi @skttl @amirkhan81 

Did this issue get solved? It's happening to me in a same way.

It works wondes on Local, where i can see the FullTextContent field - but when i updated live with same code and settings. FullTextContent field is fully empty also after reindexing more than twice. Which then makes results on search page, being different than on local.

Umbraco version 13.0.1
Full Text Search: 4.1.0.0

**Comment by skttl** [comment]
> If it's not, then try reindexing, by rightclicking a node in the content tree, and select the reindex option.

**Comment by MikeB1337** [comment]
> > If it's not, then try reindexing, by rightclicking a node in the content tree, and select the reindex option.

I don't see the reindex in "Content" nodes. Or do you mean inside the FTS "Reindex".

![Image](https://github.com/user-attachments/assets/ce82f041-f450-46c7-92d9-7a52ee810ad6)

**Comment by skttl** [comment]
Missed that you are in v13. In V13 there is a Full Text Search dashboard in settings, where you can reindex everything. If there is still no FullTextContent in the external index, check the logs. Something must have went wrong when rendering the pages.

**Comment by MikeB1337** [comment]
@skttl Makes sense now, didn't notice the log error. But it does seem like a "access"/permission issue to the live server.

![Image](https://github.com/user-attachments/assets/322503cd-5cde-4c4f-9e2e-417b6d55c49d)

Thanks for clearing that out, i will see if i can fix it somehow or with the ones who host it

**Comment by skttl** [comment]
Good to hear. If your host cant help you, then an option could be to switch to RazorPageRendering (https://github.com/skttl/umbraco-fulltextsearch8/blob/v5/dev/docs/developers-guide-v4.md#rendering) instead. Then you don't need the HTTP call.

**Comment by MikeB1337** [comment]
Awesome i will take a look at that! Thanks
